src/System.Linq/src/Var.cs

```diff
@@ -1,5 +1,4 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
-internal static class var { } // Enforce explicit declaration of variable types.
-
 
+internal static class var { } // Enforce explicit declaration of variable types.
```

Comment:
I just added a newline between the copyright and code because I'm pedantic about style. I can revert it.